econml/ortho_forest.py

```diff
@@ -1065,25 +1226,53 @@ def effect_interval(self, X=None, *, T0=0, T1=1, alpha=0.1):
         lower, upper : tuple(type of :meth:`effect(X, T0, T1)<effect>`, type of :meth:`effect(X, T0, T1))<effect>` )
             The lower and the upper bounds of the confidence interval for each quantity.
         """
-        X, T0, T1 = self._estimator._expand_treatments(*check_input_arrays(X, T0, T1))
-        dT = (T1 - T0) if T0.ndim == 2 else (T1 - T0).reshape(-1, 1)
-        params_and_cov = self._predict_wrapper(X)
-        # Calculate confidence intervals for the effect
+        eff, scales = self._effect_inference_helper(X, T0, T1)
         lower = alpha / 2
         upper = 1 - alpha / 2
-        # Calculate the effects
-        eff = np.asarray([np.dot(params_and_cov[i][0], dT[i]) for i in range(X.shape[0])])
-        # Calculate the standard deviations for the effects
-        scales = [np.sqrt(dT[i] @ params_and_cov[i][1] @ dT[i]) for i in range(X.shape[0])]
         effect_lower = eff + np.apply_along_axis(lambda s: norm.ppf(lower, scale=s), 0, scales)
         effect_upper = eff + np.apply_along_axis(lambda s: norm.ppf(upper, scale=s), 0, scales)
         return effect_lower, effect_upper
 
+    def effect_inference(self, X=None, *, T0=0, T1=1, alpha=0.1):
```

Comment:
This has the same docstring as `effect_interval`

Reply:
Good point!!